### ISSUE TITLE
fontawesome icons are not showing and logs from com.vaadin.server.VaadinServlet serveStaticResourcesInVAADIN

### ISSUE THREAD

**Issue by bedinsky** [mstahv/dawn#5]
I cannot see anymore FontAwesome icons using dawn addon installed as ivy dependency

I have added to ivy.xml
<dependency org="org.peimari" name="dawn" rev="5" />	
and removed
<dependency org="com.vaadin" name="vaadin-themes" rev="&vaadin.version;" />

why fonts are not loaded ?

Apr 14, 2015 10:28:40 AM com.vaadin.server.VaadinServlet serveStaticResourcesInVAADIN
INFO: Requested resource [/VAADIN/themes/valo/util/bourbon/css3/http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/fonts/fontawesome-webfont.ttf] not found from filesystem or through class loader. Add widgetset and/or theme JAR to your classpath or add files to WebContent/VAADIN folder.

**Comment by mstahv** [comment]
Do you use Valo theme directly or have a customised version of it? I don't see this issue with the precompiled version.

**Comment by bedinsky** [comment]
Hi ! I forgot to tell that I have my own theme 

@import "../valo/valo.scss";
...
@mixin dedalo {
	@include valo;
...

**Comment by whiteancient** [comment]
I get the same thing. The browser tells me it cant find: 
http://localhost:8042/VAADIN/themes/valo/util/bourbon/css3/http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/fonts/fontawesome-webfont.woff 

I have my own theme with 10 or so different scss-files where styles.scss looks like:

@import "activetheme";
@import .......

.activetheme {

  @include activetheme;
@include .......
}

and _activetheme.scss starts like this:

@mixin activetheme {
  @include valo;
..
..
}

This structure usually works out great, and except the fontawesome, everything else is working out fine with Dawn :)

**Comment by bedinsky** [comment]
I confirm that dawn is working great except the fontawesome !
:( menu icons are not showing

**Comment by mstahv** [comment]
I changed a bit how the font awesome is included in version 6, but still didn't try with inherited custom theme. Can you check if it works now?

**Comment by whiteancient** [comment]
I wasnt able to make it work. Now it doesnt even ask for the failing fontawesome-url.
I included this in my main scss-file:
  @include v-font(FontAwesome, "http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/fonts/fontawesome-webfont");
but it asks for the relative url "/base/fonts/xxxx.woff" etc instead. So I downloaded the fontawesome-files and put it where it is asking for it. So I have solved the error, with the drawback of not using the cdn-solution for the fontawesome-parts of Dawn.

**Comment by mstahv** [comment]
One blind guess that might solve this:

$v-relative-paths: false;

From comment: This boolean is used to flag which compiler is used, so that paths are correct for different resources. false == Ruby, true == Vaadin

The default is true, maybe using false there would avoid adding the weird prefix.

**Comment by mstahv** [comment]
@jounik, please help us :-)

**Comment by bedinsky** [comment]
I'm going to ask @jounik during live Vaadin Valo Webinar right now ;-)))

On Thu, Apr 23, 2015 at 2:25 PM, Matti Tahvonen <notifications@github.com>
wrote:

> @jounik <https://github.com/jounik>, please help us :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mstahv/dawn/issues/5#issuecomment-95568093>.
>



-- 
“Think Green - Please do not print this email unless you really need to“

**Comment by mstahv** [comment]
Yes, do that, I'm listening to it as well while hacking with something else...

**Comment by fancellu** [comment]
Argh, I've just come across this issue right now. Do tell when you find out.

**Comment by whiteancient** [comment]
I tried "$v-relative-paths: false;" in a few different places in the files. But it made no difference :(

**Comment by fancellu** [comment]
Is this bug still here for rel 7? "Built against latest 7.5-SNAPSHOT (somewhere between 7.5.0 and 7.5.1), gzipped css files for enhanced performance with Vaadin 7.5."

**Comment by mstahv** [comment]
I haven't done anything for it, and as I'm compiling with the real sass compiler I haven't faced the issue. I haven't followed the stuff they have done for the core theme, so it might or might not be fixed. If somebody tries, let us know.

Also if somebody comes up with a solution that works with vaadin sass compiler properly, I'd be happy to include the hack into dawn.

**Comment by bedinsky** [comment]
Hi,
I give a try with dawn 11 and icon issue has been fixed !
There are some minor iusses but I know they depends on my customizations.
Thank you very much !
